pkg/component/nginxingress/nginxingress.go

```diff
@@ -837,3 +874,9 @@ func (n *nginxIngress) getName(kind string, backend bool) string {
 
 	return ""
 }
+
+// GetServiceName provides the name of the service resource of the controller in the seed scenario.
```

Comment:
```suggestion
// GetServiceName provides the name of the service resource of the controller for cluster type Seed.
```

Reply:
Adapted comment in https://github.com/gardener/gardener/pull/9038/commits/5a5d31af42a290c656cfd11770ce6b8bd460add2.